src/entities/SecurityToken/Issuance/Issuance.ts

```diff
@@ -19,17 +25,7 @@ export class Issuance extends SubModule {
   /**
    * Issue a certain amount of tokens to an address. The address must already have been added via `modifyData`. Otherwise,
    * the corresponding shareholder data for that address must be supplied to this method
-   * NOTE: If shareholder data is supplied, client-side validations to verify if the transfer is possible won't be performed
-   *
-   * @param issuanceData array of issuance data
-   * @param issuanceData[].address address of the shareholder to issue tokens for
-   * @param issuanceData[].amount amount of tokens to issue
-   * @param issuanceData[].shareholderData KYC-related and other shareholder data to add/modify (optional. If not supplied, the shareholder is implied to exist already)
-   * @param issuanceData[].shareholderData.canSendAfter date after which the shareholder can transfer tokens
-   * @param issuanceData[].shareholderData.canReceiveAfter date after which the shareholder can receive tokens
-   * @param issuanceData[].shareholderData.kycExpiry date at which the shareholder's KYC expires
-   * @param issuanceData[].shareholderData.isAccredited whether the shareholder is accredited (defaults to false)
-   * @param issuanceData[].shareholderData.canBuyFromSto whether the shareholder is allowed to purchase tokens in an STO (defaults to true)
+   * **NOTE: If shareholder data is supplied, client-side validations to verify if the transfer is possible won't be performed**
```

Comment:
This ** again, I am just not sure about this as I didn't find anything like that in the docs

Reply:
markdown syntax